chrome/test/sunset-test.js

```diff
@@ -82,3 +82,33 @@ test(function () {
   test_index(Sunset.timeline_offsets_.length);
   assert_equals(reached, limit);
 }, "maybeShowNotification_ always works for large indices");
+
+test(function () {
+  // Try different configurations of local and storage data.
+  for (var local = false; local < true; local++)
```

Comment:
No, it doesn't. I tested just one case. Shame_cube << me;

Changed to ints, after public shaming :-)